yogstation/code/game/objects/items/grenades/glitterbombs.dm

```diff
@@ -0,0 +1,44 @@
+/obj/item/grenade/plastic/glitterbomb
+	name = "plastic party explosive"
+	desc = "Used to put the party in specific locations."
+	can_attach_mob = TRUE
+	var/glitter_type = "glitter"
+
+/datum/effect_system/smoke_spread/chem/glitter
+	effect_type = /obj/effect/particle_effect/smoke/chem/glitter
+
+/obj/effect/particle_effect/smoke/chem/glitter
+	lifetime = 6
+	opaque = TRUE
+
+/obj/item/grenade/plastic/glitterbomb/Initialize()
+	. = ..()
+	create_reagents(20)
+	reagents.add_reagent(glitter_type, 20)
+
+/obj/item/grenade/plastic/glitterbomb/prime()
+	var/datum/effect_system/smoke_spread/chem/glitter/smoke = new()
+	smoke.set_up(reagents, 4, target)
+	smoke.start()
+	playsound(target, 'sound/items/party_horn.ogg', 50, 1, -1)
+	target.cut_overlay(plastic_overlay, TRUE)
+
+/obj/item/grenade/plastic/glitterbomb/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] swallows [src]! It looks like [user.p_they()] WANTS TO PARTY!</span>")
```

Comment:
Why do u lie in the code? it's suicide not a party act, change it from "WANTS TO PARTY!" to "WANTS TO COMMIT SUICIDE THROUGH THE USE OF A GLITTERBOMB". Thank